db/schema.rb

```diff
@@ -96,15 +96,6 @@
 
   add_index "bounces", ["email"], name: "index_bounces_on_email", unique: true, using: :btree
 
-  create_table "claim_sources", force: :cascade do |t|
```

Comment:
Did you remove those tables manually? Because they reappeared for me when I re-ran `docker-compose up`

Reply:
I removed the migration for tables creation but there is no migration to delete existing tables (will add this migration)